test/common/quic/envoy_quic_proof_verifier_test.cc

```diff
@@ -90,6 +95,9 @@ class EnvoyQuicProofVerifierTest : public testing::Test {
   Ssl::MockCertificateValidationContextConfig cert_validation_ctx_config_;
   std::unique_ptr<EnvoyQuicProofVerifier> verifier_;
   NiceMock<Ssl::MockContextManager> tls_context_manager_;
+  Event::MockDispatcher dispatcher_;
+  Network::TransportSocketOptionsConstSharedPtr transport_socket_options_;
+  NiceMock<MockProofVerifyContext> verify_context_;
```

Comment:
Can we also add simple unit tests in this file which verify that the getters for EnvoyQuicProofVerifyContextImpl return the correct values? (If for no other reason than to keep coverage high :>)

Reply:
This test uses mocked verify context, so I wasn't able to test the real impl here. But I added a unit test in client session.